### DISCUSSION TITLE
posts/icp-filing-shit/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#42] [Comments]
# posts/icp-filing-shit/

随着在云服务器上折腾得越来越多，俺想到要给云服务器指定子域名，这样分享的链接就算以后更换云服务器厂商也不会失效，同时能将杂七杂八的东西归拢到一块儿。
[&hellip;] 操作过程是顺利的，结果是 OMG 的。建好子域名，DNS 指向云服务器的公网 IP 地址，迫不及待地尝试子域名访问，发现访问速度比 IP 访问慢很多。刷新下网页后直接被禁止访问，原因是网站未备案。
[&hellip;] 俺合计着 &hellip;

https://shitao5.org/posts/icp-filing-shit/

**Comment by yihui** [comment]
备案往事两则，供君一呵呵：

- https://yihui.org/cn/2009/06/so-much-water-in-their-brain/
- https://yihui.org/cn/2008/08/back-again/

话说 09 年你才上小学四年级或五年级吧。现在我回想起“绿坝”都觉得像是上个世纪的事了。

**Reply by Shitao5** [reply]
哈哈哈，用于备案的网站没备案，永垂青史的[滤霸・滑稽护航](https://zh.wikipedia.org/zh-cn/%E7%B6%A0%E5%A3%A9%C2%B7%E8%8A%B1%E5%AD%A3%E8%AD%B7%E8%88%AA)。现在连前几年的疫情都要叫「口罩」了，真是防民之口甚于防川。

09 年我四年级，才刚开始在~~互联网~~局域网世界摸爬滚打。历史居然重复和押韵呐，益辉二十四五岁遇到的事情，在我的二十四五岁依然会遇到。希望等我回想起「上个世纪」的事情时，有些东西被彻底丢进历史的垃圾桶，伴随着侠义之士啐上的唾沫和群众的一片叫好声。

**Comment by CyrusYip** [comment]
> 十五年过去，中文互联网似乎越来越衰微，这和备案脱不了干系。流量、内容逐渐被圈在微信、抖音等平台内部，拔网线摇身一变成了屏蔽、封号……

这里的「中文互联网」似乎用「中国大陆互联网」会更准确？中文互联网说得太广了，每次看到讲中文互联网衰落的文章，我都感觉其他使用中文的地区躺着也中枪了。

**Reply by CyrusYip** [reply]
幸好我们建了独立博客，不管外界趋势怎么变，我们还是有一个能自由写作的角落。

**Reply by Shitao5** [reply]
最近很火的那篇[《中文互联网正在加速崩塌》](https://web.archive.org/web/20240523001943/https://mp.weixin.qq.com/s/afg3zHPpEyRzSfOR1Aeh3w)用的是「中文互联网」这个词，确实是指国内，也有人叫「简体中文互联网」的。我觉得能对一个词所指有共识就可以，太准确搞得像学术写作了。

有独立博客确实是一种幸运，可以为自己写，给别人看。

**Reply by CyrusYip** [reply]
这几年知乎质量下跌很多，但它在大陆网站里相对不错。现在知乎禁止非登录用户查看完整帖子，还禁止谷歌收录，这真的是压死骆驼的最后一棵草。现在用谷歌用简体中文很容易搜到文不对题的垃圾网页，我在考虑改用繁体中文搜索。

**Reply by Shitao5** [reply]
有 GPT 之后我发现自己搜索的频率下降很多，大部分是直接问 GPT 了。现在简体中文搜出来的东西惨不忍睹，CSDN 不让复制，知乎不登录不让看……虽然有复制的插件，你也写过绕开登录的工具，但我慢慢感觉这是很「卑微」的行为，还是离这些东西远远的好，用不起还是躲得起的。

**Reply by CyrusYip** [reply]
确实是卑微的行为，用户越这样，就越受到那些糟糕网站的折磨，越是习惯这种折磨。想起了六国论里面这段话：

>今日割五城，明日割十城，然后得一夕安寝。起视四境，而秦兵又至矣。然则诸侯之地有限，暴秦之欲无厌，奉之弥繁，侵之愈急，故不战而强弱胜负已判矣。
